dfastmi/kernel/core.py

```diff
@@ -282,7 +283,7 @@ def dzq_from_du_and_h(
 
 def main_computation(
     dzq: List[numpy.ndarray],
-    T: Vector,
+    number_of_days: Vector,
     rsigma: Vector,
 ) -> Tuple[numpy.ndarray, numpy.ndarray, numpy.ndarray, List[numpy.ndarray]]:
```

Comment:
Can we do something about the return type? What does the `Tuple[numpy.ndarray, numpy.ndarray, numpy.ndarray, List[numpy.ndarray]]` represent? :D

Reply:
dzgem, dzmax, dzmin, dzb.
Nice and clean Single responsibility of the main_computation method xD

I wanted to tackle that in a seperate issue, since the 4 calls from the bedlevelcalculator can replace this method and seperate the 4 returns. (https://issuetracker.deltares.nl/browse/DFAST-93)
After discussion with Prisca we decided to leave it in main_computation for now.
This can still be tackled in the future, since batch.py also needs refactoring. (the user or this method)

Besides this, I can still put the above mentioned return type as a typehint if that would help with readability, but I do not think it will help much...

Reply:
discussed, resolve conversation.